pkg/sitewise/api/property_history.go

```diff
@@ -65,12 +73,17 @@ func historyQueryToInput(query models.AssetPropertyValueQuery) *iotsitewise.Batc
 	}
 }
 
-func BatchGetAssetPropertyValues(ctx context.Context, client client.SitewiseClient, query models.AssetPropertyValueQuery) (*framer.AssetPropertyValueHistory, error) {
+func BatchGetAssetPropertyValues(ctx context.Context, client client.SitewiseClient, query *models.AssetPropertyValueQuery) (*framer.AssetPropertyValueHistory, error) {
 	var (
 		maxDps = int(query.MaxDataPoints)
 	)
 
-	awsReq := historyQueryToInput(query)
+	err := getAndSetAssetIdAndPropertyId(query, client, ctx)
```

Comment:
Similar to other handlers, would suggest returning a query and passing around the modified query. See [here](https://github.com/grafana/iot-sitewise-datasource/pull/183/files#diff-68191c15fcbc403d2bd4bb32eb2caacf7ed821ad4d3f9f69f704e465a09d9672R71-R72)